### ISSUE TITLE
Example training log for ESRGAN training

### ISSUE THREAD

**Issue by singulart** [open-mmlab/mmsr#82]
Hi all! I would appreciate an example of training log, because I've got a feeling that in my case the model doesn't learn much: 
```
20-05-22 17:42:09.481 - INFO: [epoch:  0, iter:     600, lr:(1.000e-04,)] l_g_pix: 4.1864e-04 l_g_fea: 8.7215e-01 l_g_gan: 4.5786e-02 l_d_real: 1.1156e-04 l_d_fake: 8.2644e-05 D_real: 2.0082e+01 D_fake: 1.0925e+01
20-05-22 17:43:04.323 - INFO: [epoch:  0, iter:     650, lr:(1.000e-04,)] l_g_pix: 3.5674e-04 l_g_fea: 8.0606e-01 l_g_gan: 1.7247e-02 l_d_real: 2.0103e-02 l_d_fake: 1.9560e-02 D_real: 3.6051e+01 D_fake: 3.2641e+01
20-05-22 17:43:58.015 - INFO: [epoch:  0, iter:     700, lr:(1.000e-04,)] l_g_pix: 3.0637e-04 l_g_fea: 7.8647e-01 l_g_gan: 5.2761e-02 l_d_real: 2.8601e-05 l_d_fake: 2.1593e-05 D_real: 3.7679e+01 D_fake: 2.7127e+01
20-05-22 17:44:50.869 - INFO: [epoch:  0, iter:     750, lr:(1.000e-04,)] l_g_pix: 3.3878e-04 l_g_fea: 7.6630e-01 l_g_gan: 6.3344e-02 l_d_real: 1.2295e-05 l_d_fake: 4.8751e-06 D_real: 4.0821e+01 D_fake: 2.8152e+01
20-05-22 17:45:43.945 - INFO: [epoch:  0, iter:     800, lr:(1.000e-04,)] l_g_pix: 4.9270e-04 l_g_fea: 8.0499e-01 l_g_gan: 6.8465e-02 l_d_real: 3.5886e-06 l_d_fake: 1.4603e-06 D_real: 3.0564e+01 D_fake: 1.6871e+01
20-05-22 17:46:37.399 - INFO: [epoch:  0, iter:     850, lr:(1.000e-04,)] l_g_pix: 3.5965e-04 l_g_fea: 8.0575e-01 l_g_gan: 5.4274e-02 l_d_real: 4.2274e-05 l_d_fake: 1.6408e-05 D_real: 1.9789e+01 D_fake: 8.9344e+00
20-05-22 17:47:32.259 - INFO: [epoch:  0, iter:     900, lr:(1.000e-04,)] l_g_pix: 5.4708e-04 l_g_fea: 7.8664e-01 l_g_gan: 5.1232e-02 l_d_real: 2.3394e-05 l_d_fake: 2.1417e-05 D_real: 3.9429e+01 D_fake: 2.9183e+01
20-05-22 17:48:26.130 - INFO: [epoch:  0, iter:     950, lr:(1.000e-04,)] l_g_pix: 3.1037e-04 l_g_fea: 7.1249e-01 l_g_gan: 4.6995e-02 l_d_real: 5.5121e-05 l_d_fake: 4.9645e-05 D_real: 3.5326e+01 D_fake: 2.5927e+01
20-05-22 17:49:18.826 - INFO: [epoch:  0, iter:   1,000, lr:(1.000e-04,)] l_g_pix: 4.5797e-04 l_g_fea: 8.4661e-01 l_g_gan: 6.5273e-02 l_d_real: 1.0110e-03 l_d_fake: 1.3658e-04 D_real: 5.0026e+01 D_fake: 3.6973e+01
```
The snippet above is from the training ESRGAN from scratch using custom imageset. My validation set images are NOT a subset of those comprising the training set. Meaning that training and validation images are taken from different videos. Maybe this is the problem, I don't know. I am not entirely sure whether the metrics figures in the training log should go down.

**Comment by xinntao** [comment]
The gan loss fluctuates a lot~
It is better to check the visualization of validation images for SRGAN/ ESRGAN.

BTW, 
Thanks for using MMSR. 
We will upgrade MMSR to MMEditing (in the same repo), which consists of SR, inpainting, matting and generation tasks. 
And MMSR will be deprecated and no longer exist.
Training samples will be updated in MMEditing.
